nucypher/characters/control/specifications/fields/treasuremap.py

```diff
@@ -41,7 +41,7 @@ def _deserialize(self, value, attr, data, **kwargs):
     def _validate(self, value):
 
         splitter = BytestringSplitter(Signature,
-                                  (bytes, KECCAK_DIGEST_LENGTH),  # hrac
+                                  (bytes, 16),  # hrac
```

Comment:
Consider using a constant here instead of the literal.

Reply:
:+1: